dom.d.ts

```diff
@@ -0,0 +1,44 @@
+type Nil = null | undefined;
+type Primitive = string | number | bigint | boolean;
```

Comment:
`Primitive` also includes `Nil` and symbols. Definition:

```js
function isPrim(val) {return !isComp(val)}
function isComp(val) {return isObj(val) || isFun(val)}
```

Convenient reference: https://developer.mozilla.org/en-US/docs/Glossary/Primitive. (Also I'd use `Prim` for brevity.)